openapi/generated/lollipop_definitions.yaml

```diff
@@ -0,0 +1,108 @@
+openapi: 3.0.1
```

Comment:
I think that the validation of the new spec is failing because we still use `oval` to validate the specs [here](https://github.com/pagopa/io-backend/blob/master/.devops/code-review-pipelines.yml#L57) which does not support OpenAPI 3, we should update the pipeline with some other tool such as [swagger-cli](https://www.npmjs.com/package/@apidevtools/swagger-cli)

Reply:
Done in [[#ICC-296] change openapi validator](https://github.com/pagopa/io-backend/pull/973/commits/183d5e07df6dc958d7730dcc2dc08546a7817674)